File: Modul-6/Soal-1/PRAK601-2410817310012-FAISALTANJUNG.py
baris, kolom = map(int, input().split())
angka = list(map(int, input().split()))
matriks = []
for i in range(baris):
    matriks.append(angka[i * kolom:(i + 1) * kolom])

for baris in matriks:
    print(" ".join(map(str, baris)))

File: Modul-6/Soal-1/PRAK601-2410817310012-FAISALTANJUNG.c
#include <stdio.h>
void main(){
    int baris, kolom;
    scanf("%d %d", &baris, &kolom);

    int Matriks[baris][kolom];

    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            scanf("%d", &Matriks[i][j]);
        }
    }
    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            printf("%d ", Matriks[i][j]);
        }
        printf("\n");
    }
}